doc/man3/EVP_EncryptInit.pod

```diff
@@ -485,7 +485,7 @@ the length of the tag (with the C<tag> parameter set to NULL) when encrypting.
 The tag length is often referred to as B<M>. If not set a default value is
 used (12 for AES). When decrypting, the tag needs to be set before passing
 in data to be decrypted, but as in GCM and OCB mode, it can be set after
-passing additional authenticated data (see L<AEAD Interface>).
+passing additional authenticated data (see L<AEAD INTERFACE>).
```

Comment:
Oh my...  would you mind correcting that link while you're at it?  It should be `L</AEAD INTERFACE>`.  Here's the partial explanation from `man perlpod`:

```
       "L<name>" -- a hyperlink
           There are various syntaxes, listed below.  In the syntaxes given,
           "text", "name", and "section" cannot contain the characters '/' and
           '|'; and any '<' or '>' should be matched.

           •   "L<name>"

               Link to a Perl manual page (e.g., "L<Net::Ping>").  Note that
               "name" should not contain spaces.  This syntax is also
               occasionally used for references to Unix man pages, as in
               "L<crontab(5)>".

           •   "L<name/"sec">" or "L<name/sec>"

               Link to a section in other manual page.  E.g., "L<perlsyn/"For
               Loops">"

           •   "L</"sec">" or "L</sec>"

               Link to a section in this manual page.  E.g., "L</"Object
               Methods">"
```

Reply:
fixup commit pushed.